client/src/components/signup/signup.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import '../../containers/main.css';
+import '../common/navbar.js';
+
+export default class Signup extends Component {
+    render() {
+        return(
+            <html>
```

Comment:
This should be all wrapped inside of a `<div>` - the html and body tags are for normal webpages but in React, we already have a main html file, you just can't see it. 

This will make more sense as we go on, but for now, take out `<html><body>` and put in `<div>` instead.